### ISSUE TITLE
Supporting Leader Content Metadata

### ISSUE THREAD

**Issue by jterps08** [PowerlineApp/powerline-mobile#154]
Currently metadata (Title, Description, Image) of embedded link is only supported for posts (micropetitions). We need it to be supported for leader content. 

This requires handling on front-end, but it requires backend work adjustment too.

**Comment by petervojtek** [comment]
If I understand this task properly, we are blocked by changes to be done on backend first (and actually from frontend point of view it will be very easy to copy&paste the html from micro-petitions.

**Comment by petervojtek** [comment]
duplicate of #22 and #424
